src/System.Runtime/ref/System.Runtime.csproj

```diff
@@ -5,6 +5,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsCoreAssembly>true</IsCoreAssembly>
     <ProjectGuid>{ADBCF120-3454-4A3C-9D1D-AC4293E795D6}</ProjectGuid>
+    
+    <!-- Default interfaces are not officially supported in netcoreapp yet -->
+    <DefineConstants Condition="'$(IsPrerelease)' == 'true' and '$(TargetGroup)' == 'netcoreapp'">$(DefineConstants);FEATURE_DEFAULT_INTERFACES</DefineConstants> 
```

Comment:
Is this a netcoreapp only feature? Or is it also in uap?

Reply:
If uap = .NET Native, we haven't even started a prototype there.

Reply:
Yes. Just something to keep in mind when we start that work.